tests/test_ptc.py

```diff
@@ -314,19 +314,19 @@ def test_meanVarMeasurementAllNan(self):
     def test_getInitialGoodPoints(self):
         xs = [1, 2, 3, 4, 5, 6]
         ys = [2*x for x in xs]
-        points = self.defaultTask._getInitialGoodPoints(xs, ys, 0.1, 0.25)
+        points = self.defaultTask._getInitialGoodPoints(xs, ys, 0.1, 0.25, 0., 0.)
```

Comment:
Can these explicitly set the control parameters? `_getInitialGoodPoints(xs, ys, maxDeviationPositive=0.2, maxDeviationNegative=0.25, minMeanRatioTest=0.0, minVarPivotSearch=0.0)`?  This is easier to read and understand than a long list of floats.